x9.md

```diff
@@ -0,0 +1,57 @@
+# Status Account Creation and Maintenance Specification
```

Comment:
is there a simpler way to refer to 'creation and maintenance'? just use "Account" or 'lifecycle' or 'handling'?

Reply:
It's a lot of words already,

Status
Acconut
Creation and
Maintenance
Specification

compared to

x3dh
spec

or

double ratchet
spec

Reply:
Yes I hadn't gotten to the point of unifying all the terms. To keep it relatively general, `Account Specification` seems good.